core/src/main/java/net/opentsdb/query/filter/TagValueRangeFilterFactory.java

```diff
@@ -49,11 +48,23 @@ public QueryFilter parse(final TSDB tsdb,
     if (node == null) {
       throw new IllegalArgumentException("Node cannot be null.");
     }
-    try {
-      return (QueryFilter) mapper.treeToValue(node, TagValueRangeFilter.class);
-    } catch (JsonProcessingException e) {
-      throw new IllegalArgumentException("Failed to parse TagValueRegexFilter", e);
+    TagValueRangeFilter.Builder builder = TagValueRangeFilter.newBuilder();
+    JsonNode n = node.get("tagKey");
```

Comment:
shouldn't this be node.get("key")

Reply:
backwards compatibility. See three lines below.